pdc/apps/unreleasedvariant/views.py

```diff
@@ -0,0 +1,205 @@
+#
+# Copyright (c) 2018 Red Hat
+# Licensed under The MIT License (MIT)
+# http://opensource.org/licenses/MIT
+#
+from pdc.apps.common import viewsets
+from pdc.apps.module.models import Module
+from pdc.apps.unreleasedvariant.serializers import UnreleasedVariantSerializer
+from pdc.apps.unreleasedvariant.filters import UnreleasedVariantFilter
+
+
+class UnreleasedVariantViewSet(viewsets.PDCModelViewSet):
+    """
+    ##Overview##
+
+    This page shows the usage of the **Module API**, please see the following for more details.
+    """
```

Comment:
It would be nice to mention here that a new version is available should be preferred in new applications.

Reply:
Good point. I forgot to put that.